_config.yml

```diff
@@ -26,7 +26,7 @@ kind: "lesson"
 # Magic to make URLs resolve both locally and on GitHub.
 # See https://help.github.com/articles/repository-metadata-on-github-pages/.
 # Please don't change it: <USERNAME>/<PROJECT> is correct.
-repository: <USERNAME>/<PROJECT>
+repository: jcohen02/openrefine-socialsci
```

Comment:
It feels very counter-intuitive, and I haven't looked into how it works exactly, but you shouldn't change this line.

Reply:
Oops, my apologies, this change was, indeed, not meant to make it into the PR! Thanks for spotting this - I made the change while trying to understand the error I was getting when initially trying to run the site locally. This should be fixed now, thanks for reviewing.